tools/text_formatter.py

```diff
@@ -17,6 +17,26 @@
     "pepper-bullet": "pepper-ball",
     "protestor": "protester",
     "real-bullet": "live-round",
+    "le-lethal": "less-lethal",
+}
+
+LESS_LETHALS = {
+    "bean-bag",
+    "foam-bullet",
+    "marking-round",
+    "tear-gas",
+    "pepper-spray",
+    "mace",
+    "paintball",
+    "pepper-ball",
+    "projectile",
+    "rubber-bullet",
+    "stun-grenade",
+    "taser",
+    "tear-gas",
+    "tear-gas-canister",
+    "wooden-bullet",
+    "lrad",
 }
```

Comment:
How would we feel about a new enhancement Issue to collapse these tag-related lists & the `possible_tags.md` file into one?

Or even an Issue to autogenerate the `possible_tags.md` file for the Reporters to more easily read, yet keep the various tagging lists in 1 place/1 file.

Reply:
Yeah I've thought about it before and we probably _should_ but I just haven't thought of a great way. but definitely a place we can improve

Reply:
@emarcey just updated ticket #1775 to reflect this convo, gave it some direction/a goal. Will probably pause the zip ties work at some point soon and look at that issue, instead of waiting for completion, b/c I'm just about wrapping up _California_ (this state took a while + I've been busy with other stuff).